src/sentry/templates/sentry/layout.html

```diff
@@ -32,6 +32,9 @@
 
   {% block scripts %}
   <script src="{% asset_url "sentry" "dist/vendor.js" %}"{% crossorigin %}></script>
+  {% ifnotequal request.LANGUAGE_CODE "en" %}
+  <script src="{% locale_js_url request %}"{% crossorigin %}></script>
```

Comment:
When `locale_js_url` spits out `None` (when the requested language isn't a language we support), what gets spit out into the HTML? I assume this will be bad since we'd get at best:

`<script src=""></script>`

Which will cause the current page to be loaded and attempted to be executed as a script.